module/planning/PlanKick/src/PlanKick.cpp

```diff
@@ -88,9 +88,9 @@ namespace module::planning {
                     return;
                 }
 
-                // If the kick leg is forced left, kick left
-                // If the kick leg is auto, kick with left leg if ball is more to the left
-                if (cfg.kick_leg == LimbID::LEFT_LEG || (cfg.kick_leg == LimbID::UNKNOWN && ball.rBTt.y() > 0.0)) {
+                // If the kick leg is forced left, kick left If the kick leg is auto,
+                // kick with left leg if ball is more to the left
```

Comment:
either put a full stop at the end of the first sentence or put the second sentence on the next line